### PR TITLE
chore(apple-pay): add error message to apple pay session failure

### PR DESCRIPTION
**Pull request by douglaseggleton**
# Description

Captures error messages for failing to start an Apple Pay session.

## Contribution guidelines

For contribution guidelines, styleguide, and other helpful information please
see the `CONTRIBUTING.md` file in the root of this project.

### CODE REVIEW

File: packages/embed/src/subjects.ts [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const createSubjectManager = () => {
     applePayAuthorized$: createSubject<ApplePayJS.ApplePayPaymentToken>(),
     appleCompletePayment$: createSubject<boolean>(),
     appleAbortSession$: createSubject(),
-    appleSessionError$: createSubject(),
+    appleSessionError$: createSubject<string>(),
```
Comment (cbetta) [inline packages/embed/src/subjects.ts:42]: @douglaseggleton is this something a merchant can listen to? If so, does this run the risk of anything breaking of they already listen to this event?

Comment (douglaseggleton) [reply packages/embed/src/subjects.ts:42]: This is an internal event. It will go to Embed UI.

Comment (cbetta) [reply packages/embed/src/subjects.ts:42]: Tnx. Have a great trip!

**Review by cbetta — APPROVED** [review]

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.24.4`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.24.4) :rocket:

**Pull request merged**